frontend/src/components/questions/UploadQuestion.vue

```diff
@@ -41,22 +54,17 @@ function onDragLeave(event: DragEvent) {
     isDragOver.value = false;
 }
 
-function onFileAdd(event: HTMLInputEvent | DragEvent) {
+function onFileAdd(event: Event) {
     event.preventDefault();
-    const files = getFiles(event);
-    if (files?.length) {
-        for (const file of files) {
-            if (filesList.value.includes(file)) {
-                removeFileFromList(filesList.value.indexOf(file));
-            }
-            if (filesList.value.length < props.spec.max_files) {
-                // TODO: Check file size too
-                filesList.value.push(file);
-            }
-        }
-        // TODO: Process the files in filesList as needed (maybe using a helper func defined in libs)
+    const fileList = getFileList(<HTMLInputEvent | DragEvent>event);
```

Comment:
This looks like a magic trick 😂 if it works, it's ok for me

Reply:
It didn't, apparently. I used the old `as`